### DISCUSSION TITLE
Error when building v1.2.10271 in Visual Studio

### DISCUSSION THREAD

**Discussion by vikekh** [microsoft/winget-cli#2098] [Q&A]
I experience problems building the solution with tag `v1.2.10271`. I get errors like the following:

```
Specified EntryPointExe 'winget.exe' was not found in the project outputs.
```

I have the prerequisites:

- Windows 10 1809 (17763) or later: **Windows 10 Pro v19044.1645**
- [Developer Mode enabled](https://docs.microsoft.com/windows/uwp/get-started/enable-your-device-for-development)
- [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/): **Visual Studio Community 2019 v16.11.12**
- The following workloads:
  - .NET Desktop Development
  - Desktop Development with C++
  - Universal Windows Platform Development
- The following extensions:
  - [Microsoft Visual Studio Installer Projects](https://marketplace.visualstudio.com/items?itemName=VisualStudioClient.MicrosoftVisualStudio2017InstallerProjects)

I have gotten it to build on a previous version on a previous installation. Any ideas? Not very well-versed in VS C++, only C#/ASP.NET.

**Comment by denelon** [comment]
Is this still happening after:
* https://github.com/microsoft/winget-cli/pull/2156

**Comment by ThatsJustCheesy** [comment]
For any future searchers, I had the same error message, but after a bunch of these warnings:

> Skipped […] Project not selected to build for this solution configuration

I had to **select a configuration other than the default "Any CPU"** (e.g., "x64") at the top of the Visual Studio window.
